tools/augurpy/README.rst

```diff
@@ -0,0 +1,67 @@
+augurpy
```

Comment:
You can remove most of it. It would be much better if you added some developer information here like:

"Augurpy implemented after paper 1 and paper 2 and repository X. Has limitation X Y and Z. Implementation was realized through interesting facts 1 2 and 3

Reply:
Does the file stay though? Or should this be added to pertpys README?